packages/payment/src/models/payment-provider.ts

```diff
@@ -0,0 +1,12 @@
+import { Entity, PrimaryKey, Property } from "@mikro-orm/core"
+
+@Entity({ tableName: "payment-provider" })
```

Comment:
todo: _snake_casify_
```suggestion
@Entity({ tableName: "payment_provider" })
```